src/components/PeopleRender.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { PersonRender } from './PersonRender';
+import { EmptyTableMessage } from './EmptyTableMessage';
+
+import { Person } from '../types';
+import { Link } from 'react-router-dom';
+import { useFilters } from '../hooks/useFilters';
+
+interface Props {
+  people: Person[];
+}
+
+const SORT_ORDERS = ['Name', 'Sex', 'Born', 'Died'];
```

Comment:
Using an enum for `SORT_ORDERS `would be more robust and type-safe. Enums make it clear that these are predefined values for sorting and provide better autocompletion and error checking in TypeScript.